build-ps/percona-xtradb-cluster.spec

```diff
@@ -906,11 +907,16 @@ if [ -x %{_sysconfdir}/init.d/mysql ] ; then
 fi
 
 %post -n Percona-XtraDB-Cluster-server%{product_suffix}
-
 if [ X${PERCONA_DEBUG} == X1 ]; then
         set -x
 fi
 
+ver=`/bin/rpm -qi percona-xtrabackup | grep Version | awk '{print $3}'`
+if [ $ver = '2.3.5' ]; then
```

Comment:
@tplavcic yes sure I would update the version check. But what about the solution in general?
Is it acceptable or not?